examples/UCI_benchmark_intRVFL_GLVQ.py

```diff
@@ -0,0 +1,304 @@
+        "wall-following": (1200, 3, 15, 1),
+        "waveform": (1400, 7, 15, 1),
+        "waveform-noise": (1300, 15, 14, 1),
+        "wine": (850, 1, 15, 1),
+        "wine-quality-red": (1100, 1, 13, 1),
+        "wine-quality-white": (950, 3, 14, 1),
+        "yeast": (1350, 1, 5, 1),
+        "zoo": (400, 7, 1, 1),
+    }
+
+    def __init__(
+        self,
+        dataset: torchhd.datasets.CollectionDataset,
+        num_feat: int,
+        max_epochs: int = 100,
+        device: torch.device = None,
+    ):
+        super(IntRVFLGLVQ, self).__init__()
+        self.device = device
+        self.num_feat = num_feat
+        self.max_epochs = max_epochs
+        # Fetch the hyperparameters for the corresponding dataset
+        hyper_param = self.INT_RVFL_GLVQ_HYPER[dataset.name]
+        # Dimensionality of vectors used when transforming input data
+        self.dimensions = hyper_param[0]
+        # Parameter of the clipping function used as the part of transforming input data
+        self.kappa = hyper_param[1]
+        self.transfer_beta = hyper_param[2]
+        self.prototypes_per_class = hyper_param[3]
+        # Number of classes in the dataset
+        self.num_classes = len(dataset.classes)
+        # Set up the encoding for the model as specified in "Density"
+        self.hypervector_encoding = torchhd.embeddings.Density(
+            self.num_feat, self.dimensions
+        )
+
+    # Specify encoding function for data samples
+    def encode(self, x):
+        return self.hypervector_encoding(x).clipping(self.kappa)
+
+    # Specify how to make an inference step and issue a prediction
+    def forward(self, x):
+        # Make encodings for all data samples in the batch
+        encodings = self.encode(x)
+        # Classify with GLVQ classifier
+        predictions = self.classifier.predict(encodings)
+        return predictions
+
+    # Train the classfier
+    def fit(
+        self,
+        train_ld: torch.utils.data.dataloader.DataLoader,
+    ):
+        # Hyperparameters for GLVQ
+        hparams = dict(
+            distribution={
+                "num_classes": self.num_classes,
+                "per_class": self.prototypes_per_class,
+            },
+            transfer_beta=self.transfer_beta,
+            lr=0.1,
+        )
+        # Initialize the GLVQ classifier
+        self.classifier = GLVQ(
+            hparams,
+            optimizer=torch.optim.Adam,
+            prototypes_initializer=pt.initializers.SMCI(train_loader.dataset),
+            lr_scheduler=ExponentialLR,
+            lr_scheduler_kwargs=dict(gamma=0.99, verbose=False),
+        )
+
+        # Setup trainer
+        trainer = pl.Trainer(
+            callbacks=[
+                TQDMProgressBar(refresh_rate=3),
+            ],
+            max_epochs=self.max_epochs,
+            detect_anomaly=True,
+        )
+
+        # Training loop
+        trainer.fit(self.classifier, train_ld)
```

Comment:
I got an error when running this example:
```
File "../site-packages/prototorch/models/abstract.py", line 189, in log_acc
    accuracy = torchmetrics.functional.accuracy(preds.int(), targets.int())
TypeError: accuracy() missing 1 required positional argument: 'task'
```
It looks like `prototorch_models` is relying on an older version of `torchmetrics` which recently changed their API design. Downgrading to `torchmetrics=0.10.3` worked for me.